todolist/todo/api.py

```diff
@@ -1,23 +1,120 @@
 from django.contrib.auth.models import User
 from tastypie import fields
-from tastypie.resources import ModelResource
+from tastypie.resources import ModelResource, Resource, ALL, ALL_WITH_RELATIONS
 from todo.models import Task
 from tastypie.authentication import SessionAuthentication
+from tastypie.authorization import DjangoAuthorization,Authorization
+from django.contrib.auth import authenticate, login, logout
+from tastypie.http import HttpUnauthorized, HttpForbidden
+from django.conf.urls import  url
+from tastypie.utils import trailing_slash
+from django.core.exceptions import ObjectDoesNotExist
+from authorization import UsersAuthorization, TasksAuthorization
 
 
+###################User resource####################
+
 class UserResource(ModelResource):
     class Meta:
         queryset = User.objects.all()
-        resource_name = 'users'
+        resource_name = 'user'
```

Comment:
why do you use here `user` and `tasks` below as resource name? Could you please be consistent